mmv1/third_party/terraform/website/docs/r/compute_router_peer.html.markdown

```diff
@@ -316,6 +335,8 @@ The following arguments are supported:
 * `project` - (Optional) The ID of the project in which the resource belongs.
     If it is not provided, the provider project is used.
 
+* `md5_authentication_key` - (Optional) Present if MD5 authentication is enabled for the peering.
```

Comment:
nit: I think we should deviate from the REST docs on this one based on terminology in https://cloud.google.com/network-connectivity/docs/router/how-to/use-md5-authentication#add-authentication, as the current description fits a proto file and not a REST API (much less a Terraform configuration). I'd suggest something like:

`Configuration for MD5 authentication on the BGP session. Structure is [documented below](#nested_md5_authentication_key).`

Reply:
Ack.